lib/generators/rspec/system/system_generator.rb

```diff
@@ -0,0 +1,25 @@
+require 'generators/rspec'
+
+module Rspec
+  module Generators
+    # @private
+    class SystemGenerator < Base
+      class_option :system_specs, :type => :boolean, :default => true,  :desc => "Generate system specs"
+
+      def generate_system_spec
+        return unless options[:system_specs]
+        raise "System Tests are available for Rails >= 5.1" if ::Rails::VERSION::STRING < '5.1'
```

Comment:
Is it what we want? 🤔 

I look at other parts of RSpec code and didn't see that kind of behavior implemented.

Reply:
The generator shouldn't be available on old rails...

Reply:
So we should have a higher condition if not on > Rails 5.1?

Reply:
I don't think we should even define it we can't run it.